lib/view/profiel/ProfielForm.php

```diff
@@ -128,19 +128,20 @@ public function __construct(Profiel $profiel, $inschrijven) {
 
 			$fields[] = new HtmlComment($html);
 		}
-
+		$fields[] = new Subkopje('Identiteit');
 		if ($admin OR $inschrijven OR $profiel->isOudlid()) {
-			$fields[] = new Subkopje('Identiteit');
 			$fields[] = new RequiredTextField('voornaam', $profiel->voornaam, 'Voornaam', 50);
 			$fields[] = new RequiredTextField('voorletters', $profiel->voorletters, 'Voorletters', 10);
 			$fields[] = new TextField('tussenvoegsel', $profiel->tussenvoegsel, 'Tussenvoegsel', 15);
 			$fields[] = new RequiredTextField('achternaam', $profiel->achternaam, 'Achternaam', 50);
+		}
+		$fields[] = new TextField('nickname', $profiel->nickname, 'Bijnaam', 20);
+		if ($admin OR $inschrijven OR $profiel->isOudlid()) {
```

Comment:
sonar miept over or, maar dat is beter om even los te fixen door het hele bestand.